### PR TITLE
SC-7318: security manager permissions

### PR DESCRIPTION
**Pull request by rabbitstack**

### CODE REVIEW

File: docs/agents/sematext-agent/jmx-attaching-containers.md [hunk 1]
```diff
@@ -19,3 +19,8 @@ There are cases where automatic attaching may not be desirable:
 If you don't want the service monitored at all just remove `MONITORING_TOKEN` env variable from its container. If you want it to be monitored, but using your
 own JMX authentication definition (either existing one or the one you will define just for this), [define container jmx settings manually](../../monitoring/manually-setting-jmx-containers).
 
+### Security manager permissions
+
+When running your JVM applications under the supervision of the security manager, you'll have to register the permission to allow Sematext Agent bootstrap the JMX connector in the target process. For this purpose, add the following permission directive in your policy file:
+
```
Comment (bsmid) [inline docs/agents/sematext-agent/jmx-attaching-containers.md]: I think this is a general note, not just related to containers (attaching in BMVM is close), so we probably need to put it elsewhere. Additionally, add a note to our FAQ document where we show the kind of error one may get and link to  your paragraph as one possible solution. Finally, might be useful to mention Solr 9.0 in that FAQ note as one example.

Comment (rabbitstack) [reply docs/agents/sematext-agent/jmx-attaching-containers.md]: OK, I could move it to the FAQ section, but note this info is exclusively related to auto attaching (this we should probably rename `jmx-attacing-containers.md` to `jmx-attaching.md`).

Comment (bsmid) [reply docs/agents/sematext-agent/jmx-attaching-containers.md]: I did't mean move this to FAQ. I would add to FAQ a separate entry which people may find when they have issues. That entry would link to this new paragraph you wrote (which would be moved to jmx attaching doc that isn't just about containers).

Comment (rabbitstack) [reply docs/agents/sematext-agent/jmx-attaching-containers.md]: @bsmid check now

File: docs/agents/sematext-agent/jmx-attaching.md [hunk 2]
```diff
@@ -19,3 +19,12 @@ There are cases where automatic attaching may not be desirable:
 If you don't want the service monitored at all just remove `MONITORING_TOKEN` env variable from its container. If you want it to be monitored, but using your
 own JMX authentication definition (either existing one or the one you will define just for this), [define container jmx settings manually](../../monitoring/manually-setting-jmx-containers).
 
+### Security manager permissions
+
+When running your JVM applications under the supervision of the security manager, you'll have to register the permission to allow Sematext Agent bootstrap the JMX connector in the target process. For this purpose, add the following permission directive in your policy file:
```
Comment (adnanrahic) [inline docs/agents/sematext-agent/jmx-attaching.md]: permission to allow Sematext Agent bootstrap the JMX connector
=>
permission to allow Sematext Agent to bootstrap the JMX connector

**Pull request merged**